### PR TITLE
Use external dependencies repo

### LINKED ISSUE #22: Use external dependencies repo
Further deprecating chroma-externals, use the Copr repo for externals during installation/test.

This means removing these dependencies from the MFL build.

### PR DESCRIPTION
**Pull request by brianjmurrell**
Fixes #22

Further deprecating chroma-externals, use the Copr repo for external
during installation/test.

Add dependencies to be built by Copr.

Roll in some .gitignore updates

### CODE REVIEW

File: chroma-bundles/include/find_deps.py [hunk 1]
```diff
@@ -61,33 +62,40 @@ def req2pkgs(self, req):
...
                             self.missing_deps.append(npkg.name)
             else:
                 if name.find(' ') > -1:
                     name = name[0:name.find(' ')]
-                self.missing_deps.append(name)
+                if name not in self.all_pkg_names:
+                    print >>sys.stderr, "adding to missing_deps 2"
+                    self.missing_deps.append(name)
 
+        #print "missing_deps: %s:" % self.missing_deps
```
Comment (tanabarr) [inline chroma-bundles/include/find_deps.py]: not sure we should be adding a comment out line?

File: chroma-dependencies/Makefile [hunk 2]
```diff
@@ -41,6 +48,20 @@ requirements::
...
+	echo "nose==1.2.1";                                       \
+	echo "supervisor==3.0b1";                                 \
+	echo "pyparsing==1.5.6";                                  \
+	echo "kombu==3.0.19";                                     \
+	echo "jsonschema==0.8.0";                                 \
+	echo "nose-testconfig==0.8";                              \
+	echo "mock==1.0.1";                                       \
+	echo "gunicorn==18.0";                                    \
+	echo "pytz==2014.4";                                      \
+	echo "https://dl.dropboxusercontent.com/s/qqjud11bvh8vo68/coverage-3.5.2pl2.tar.gz"; \
```
Comment (jgrund) [inline chroma-dependencies/Makefile]: Should we still have this dropbox file?

File: chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py [hunk 3]
```diff
@@ -0,0 +1,361 @@
+# -*- coding: utf-8 -*-
```
Comment (jgrund) [inline chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py]: Was this meant to be committed?

File: chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup [hunk 4]
```diff
@@ -125,7 +100,7 @@ retries=50
 timeout=180
 EOF
     $PROXY yum install -y python-setuptools
-    $PROXY yum install -y --disablerepo=* --enablerepo=chroma python-coverage
+    $PROXY yum install -y python2-coverage
```
Comment (tanabarr) [inline chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup]: is there a need to change the package name?

File: chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py (hunk 3 above)
Comment (tanabarr) [inline chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py]: is this not part of an external dependency that be installed from package?

File: chroma-dependencies/Makefile (hunk 2 above)
Comment (brianjmurrell) [inline chroma-dependencies/Makefile]: Yup.  We patch coverage.  Don't really know why, but we do.

File: chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py (hunk 3 above)
Comment (brianjmurrell) [inline chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py]: Strange.  I wonder how this got added.

File: chroma-agent/chroma-agent.spec [hunk 5]
```diff
@@ -24,7 +24,8 @@ Requires: python-argparse
 Requires: python-daemon
 Requires: python-setuptools
 Requires: python-requests >= 2.6.0
-Requires: python-tablib yum-utils
+Requires: python2-tablib
```
Comment (brianjmurrell) [inline chroma-agent/chroma-agent.spec]: It's part of Red Hat's transition plan to Python 3.  All Python packages going forward have the major version number of the Python they are being built for.  Eventually (i.e. as they are rebuilt) all python-* packages will become python{2,3}-*.

File: chroma-bundles/include/find_deps.py (hunk 1 above)
Comment (brianjmurrell) [inline chroma-bundles/include/find_deps.py]: It's frequently needed when debugging dependencies and I was getting tired of having to keep adding it.

File: chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup (hunk 4 above)
Comment (brianjmurrell) [inline chroma-manager/tests/framework/integration/existing_filesystem_configuration/cluster_setup]: As above.

File: chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py (hunk 3 above)
Comment (brianjmurrell) [inline chroma-manager/chroma_test_env/lib/python2.7/site-packages/pip/vendor/distlib/manifest.py]: As above..

**Pull request merged**